homework/style.css

```diff
@@ -1,3 +1,119 @@
+* {
```

Comment:
On a wide-screen monitor, your UI looks a bit stretched out. Perhaps set a `max-width` and horizontally center the content?

![image](https://user-images.githubusercontent.com/2788771/65888259-83ad7900-e39f-11e9-97f4-3510db45f1f9.png)